test/e2e/scenario/eventBridge.scenario.ts

```diff
@@ -91,4 +91,14 @@ describe('bridge present', () => {
       expect(intakeRegistry.logsEvents.length).toBe(1)
       expect(intakeRegistry.hasOnlyBridgeRequests).toBe(true)
     })
+
+  createTest('send records to the bridge')
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** what about adding a test where to ensure that in bridge mode stop / start recording also work as expected?
‏